src/web/Clipboard.ts

```diff
@@ -6,6 +6,7 @@
 *
 * Web-specific implementation of the cross-platform Clipboard abstraction.
 */
+import _ = require('lodash');
```

Comment:
You can add `escape` to **lodashMini** _(src/common/lodashMini.ts)_ and import it from **lodashMini** instead of import entire **lodash**., or import only `escape`, like so

```typesscript
import escape = require('lodash/escape')
```
or 
```typesscript
import { escape } from 'common/lodashMini';
```